Automattic/wp-calypso
# Gutenframe: Add a denied list of plugins that will not allow Gutenframe.

#### Changes proposed in this Pull Request

* Add the `isAnyPluginActive` selector to determine if any of the denied plugins are active on the current site.
* If so, Gutenframe will fall back to a standard redirect to the `wp-admin` block editor.

#### Testing instructions

* Install a denied plugin on a test JP site.
* Load the site at `/block-editor/post/:site`, and verify you are forwarded instead to the site's WP Admin block editor.

Fixes #32887.
Fixes #32934.
